build.sbt

```diff
@@ -322,6 +355,24 @@ lazy val skipReleaseSettings = Seq(
   publish / skip := true
 )
 
+/**
+ * Release settings for artifact that contains only Java source code
+ */
+lazy val javaOnlyReleaseSettings = releaseSettings ++ Seq(
+  // drop off Scala suffix from artifact names
+  crossPaths := false,
+
+  // we publish jars for each scalaVersion in crossScalaVersions. however, we only need to publish
+  // one java jar. thus, only do so when the current scala version == default scala version
+  publishArtifact := {
+    val (expMaj, expMin, _) = getMajorMinorPatch(default_scala_version)
+    s"$expMaj.$expMin" == scalaBinaryVersion.value
```

Comment:
Does this not work with just `scalaVersion` instead of `scalaBinaryVersion` without the min/maj parsing?

Reply:
👀  hm. this is what we were doing in the connectors repo ... so that's what I did here.